src/editor/Gui.cpp

```diff
@@ -3125,8 +3124,7 @@ void Gui::drawLightMapTool() {
 	static Texture* currentlightMap[MAXLIGHTMAPS] = { nullptr };
 	static float colourPatch[3];
 
-	int readLightmaps = MAXLIGHTMAPS;
-	int windowWidth = 50 + readLightmaps > 1 ? 500 : 200;
+	int windowWidth = 50 + MAXLIGHTMAPS > 1 ? 500 : 200;

```

Comment:
`MAXLIGHTMAPS > 1` is always true, but the selected face might not have more than 1 lightmap. You can use `Bsp::lightmap_count(int faceIdx)` if you want to make this dynamic.

Reply:
Ok